backtracking/minimax.py

```diff
@@ -16,6 +16,21 @@ def minimax(
     depth: int, node_index: int, is_max: bool, scores: list[int], height: float
 ) -> int:
     """
+    This function implements the minimax algorithm,
+    which helps achieve the maximum score in a game
+    by checking all possible moves.
```

Comment:
```suggestion
    This function implements the minimax algorithm, which helps achieve the optimal
    score for a player in a two-player game by checking all possible moves. If the player
    is the maximizer, then the score is maximized. If the player is the minimizer, then
    the score is minimized.
```
Clarifying the documentation